subiquity/models/subiquity.py

```diff
@@ -435,7 +435,10 @@ def _cloud_init_files(self):
                         "metadata": metadata,
                     },
                 },
-            }
+            },
```

Comment:
This all looks fine Chad, thanks!
While we're here, do you mind fact-checking the above TODO?

Reply:
@dbungert nice. looking to drop this comment, I'll mention it in the commit message too. This historic comment was from a point in time 2017 36bcc539c when cloud-init was only seeding /var/lib/cloud/seed/nocloud-net/(meta-data|user-data) files via write_files in the target in POST_INSTALL_CONFIG.  This is no longer relevant as subiquity is providing DataSourceNone configuration now with merged user_data, also sets `datasource_list: [None]` for first boot and writes `/etc/cloud/cloud-init.disabled` to prevent cloud-init from ever running on subsequent boots without someone first running `cloud-init clean` which will remove subiquity's supplemental /etc/cloud/cloud-init.disabled flag file and the 99-installer.cfg which limits datasource_list to None.

Reply:
Excellent, I appreciate it.